episodes/what-is-an-image.md

```diff
@@ -0,0 +1,540 @@
+For example, our mitosis image uses 8 bits to store each value (i.e. 
+a series of 8 ones or zeros like `00000000`, or `01101101`...).
+
+The reason the bit depth is so important is that it dictates the number of 
+different values that can be stored. In fact it is equal to:
+
+\[\large \text{Number of values} = 2^{\text{bit depth}}\]
+
+Going back to our mitosis image, this means that it can store $2^8 = 256$ 
+different values, which is equal to a range of 0-255 for unsigned integers.
+
+We can verify this by looking at the maximum value of the mitosis image:
+
+
+```python
+
+print(image.max())
+```
+
+```output
+
+255
+```
+
+You can also see this by hovering over the brightest nuclei in the viewer and 
+examining their pixel values. Even the brightest nuclei won't exceed the limit 
+of 255.
+
+::::::::::::::::::::::::::::::::::::: challenge 
+
+## Dimensions and data types
+
+Let's open a new image by copying and pasting the following lines into the 
+Napari console:
+
+```python
+
+from skimage import data
+viewer.add_image(data.brain()[9, :, :], name="brain")
+image = viewer.layers["brain"].data
+
+```
+
+This opens a new 2D image of part of a human head X-ray.
+
+- What are the dimensions of this image?
+
+- What type and bit depth is this image?
+
+- What are the possible min/max values of this image array, based on the 
+bit depth?
+
+:::::::::::::::::::::::: solution 
+
+### Solution
+
+### Dimensions
+
+The image's dimensions are (256, 256)
+
+```python
+
+image.shape
+```
+
+```output
+
+(256, 256)
+```
+
+### Type and bit depth
+
+The image's type and bit depth are: unsigned integer 16-bit
+
+```python
+
+image.dtype
+```
+
+```output
+
+dtype('uint16')
+```
+
+### Min and max
+
+Based on a bit depth of 16, this image can store $2^{16} = 65536$ values. As it 
+is of type 'unsigned integer' this corresponds to a min and max of 0 and 65535.
+
+:::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+## Common data types
+
+NumPy supports a [very wide range of data types](
+https://numpy.org/doc/stable/reference/arrays.scalars.html#sized-aliases), 
+but there are a few that are most common for image data:
+
+| NumPy datatype     | Full name                |
+| :---------         | :--------------          |
+| `uint8`            | Unsigned integer 8-bit   |
+| `uint16`           | Unsigned integer 16-bit  |
+| `float32`          | Float 32-bit             |
+| `float64`          | Float 64-bit             |
+
+`uint8` and `uint16` are most common for images from light microscopes. 
+`float32` and `float64` are common during image processing (as we will see in 
+later episodes).
+
+## Choosing a bit depth
+
+Most images are either 8-bit or 16-bit - so how to choose which to use? 
+A higher bit depth will allow a wider range of values to be stored, but also 
+result in larger image files overall. In general, a 16-bit image will have a 
```

Comment:
```suggestion
A higher bit depth will allow a wider range of values to be stored, but it will also 
result in larger file sizes for the resulting images. In general, a 16-bit image will have a 
```